README.md

```diff
@@ -205,6 +211,19 @@ The Diffie-Hellman group should have at least 1024 bit and can be created with t
 ```
 # openssl dhparam -out /etc/exim4/dhparam.pem 2048
 ```
+In the transport section, you need to modify the user and group to the exim-user `system_aliases`-transport (around line 800):
+```
+system_aliases:
+  driver = redirect
+  allow_fail
+  allow_defer
+  data = ${lookup{$local_part}lsearch{/etc/aliases}}
+  user = <exim-user>
+  group = <exim-group>
+  file_transport = address_file
+  pipe_transport = address_pipe
```

Comment:
1. Then you should fix the first sentence.
2. Surely redirecting root mail via /etc/aliases is convenient, but how are these last four lines important for this? Like I said, this works fine in Debian, even though I don't have any of `SYSTEM_ALIASES_USER`, `SYSTEM_ALIASES_GROUP`, `SYSTEM_ALIASES_FILE_TRANSPORT`, `SYSTEM_ALIASES_PIPE_TRANSPORT` or `SYSTEM_ALIASES_DIRECTORY_TRANSPORT` defined.